src/ProjectModule/components/ProjectList/ProjectList.tsx

```diff
@@ -13,6 +15,17 @@ interface progectListType {
 }
 
 export default function ProjectModule() {
+
+    // dark Light moda
+    const darkModeContext = useDarkMode();
+
+    if (!darkModeContext) {
+      return null;
+    }
+  
+    const { isDarkMode, toggleDarkMode } = darkModeContext;
+  // 
+  
```

Comment:
### This comment is for Line 102

🚀 It's a good practice to use `useCallback` for functions passed as dependencies to `useEffect`.